src/elements/content-sidebar/SidebarNavSign.tsx

```diff
@@ -17,48 +15,43 @@ import { Menu, MenuItem } from '../../components/menu';
 import messages from './messages';
 
 import './SidebarNavSign.scss';
+// @ts-ignore Module is written in Flow
+import type { TargetingApi } from '../../features/targeting/types';
+
+export interface SignSideBarProps {
+    blockedReason: string;
+    enabled: boolean;
+    onClick: () => void;
+    onClickSignMyself: () => void;
+    targetingApi: TargetingApi | null;
```

Comment:
`targetingApi` is an optional prop in `SidebarNavSignButton`. can this line align with that?

```suggestion
    targetingApi?: TargetingApi;
```